test/cell_test.rb

```diff
@@ -0,0 +1,107 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def test_it_exists
+    cell = Cell.new("B4")
+
+    assert_instance_of Cell, cell
+  end
+
+  def test_it_starts_with_a_coordinate_and_empty
+    cell = Cell.new("B4")
+
+    assert_equal "B4", cell.coordinate
+    assert_nil = cell.ship
```

Comment:
Not sure I understand why you're using assert_nill maybe we can talk about that

Reply:
![image](https://user-images.githubusercontent.com/58127453/73774907-10978300-4742-11ea-88bc-2f7a8ae31136.png)

The pry workflow has it in there as wanting it to return nil.  I thought it was weird too but it should work...

Reply:
And I know it's not called "pry workflow" but I can't ever remember what the word is....